cli.js

```diff
@@ -93,11 +94,21 @@ if (cli.flags.init) {
 	return;
 }
 
+var nodePaths;
+if (process.env.NODE_PATH) {
+    nodePaths = process.env.NODE_PATH.split(path.delimiter).map(function (p) {
+        return path.resolve(process.cwd(), p);
```

Comment:
Just realized that `path.resolve(p)` defaults to `path.resolve(process.cwd(), p)` if `p` is relative. In other words you don't need to provide `process.cwd()` here :boom: